specification/peering/resource-manager/Microsoft.Peering/preview/2019-03-01-preview/peering.json

```diff
@@ -0,0 +1,1309 @@
+        "operationId": "PeeringLocations_List",
+        "parameters": [
+          {
+            "name": "kind",
+            "in": "query",
+            "description": "The kind of the peering.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "Direct",
+              "Exchange"
+            ]
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PeeringLocationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation has failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List direct peering locations": {
+            "$ref": "./examples/ListDirectPeeringLocations.json"
+          },
+          "List exchange peering locations": {
+            "$ref": "./examples/ListExchangePeeringLocations.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Peering/peerings/{peeringName}": {
+      "get": {
+        "tags": [
+          "Peerings"
+        ],
+        "description": "Gets an existing peering with the specified name under the given subscription and resource group.",
+        "operationId": "Peerings_Get",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "peeringName",
+            "in": "path",
+            "description": "The name of the peering.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Peering"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation has failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a peering": {
+            "$ref": "./examples/GetPeering.json"
+          }
+        }
+      },
+      "put": {
```

Comment:
>put [](start = 7, length = 3)

this is not marked as long running. So, I would presume this api (PUT) completes synchronously.  can you confirm ?

Reply:
we fixed this. you were correct.

Reply:
Yes, this operation completes synchronously with ProvisioningState "Succeeded". We track the actual peering state under a different variable -- "ConnectionState" -- within the properties.